hw3/style.css

```diff
@@ -0,0 +1,50 @@
+header{
+    background-color: red;
+    color: white;
+    font-weight: bold;
+    font-size: 30px;
+    text-align: center;
+    width: 760px;
+    margin: auto;
+    margin-bottom: 20px;
+}
+img.sunset{
+    margin-left: auto;
+    margin-right: auto;
+    display: block;
+}
+div{
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    height: 100%; 
+}
+.car{
+    border-style: dashed;
+    margin-right: 4px;
+}
+.italic{
+    font-style: italic;
+}
+.link{
+    text-decoration: none;
+}
+p{
+    width: 760px;
+    margin: auto;
+    margin-bottom: 20px;
+    margin-top: 20px; 
+    font-size: 18px;
+}
+span:before{
+    content: "\00A9 \00A0";
+}
+footer{

```

Comment:
better to use class selector, not a tag - minor

Reply:
done